docs/development/testguide.rst

```diff
@@ -212,10 +231,10 @@ For more information on the ``pytest-openfiles`` plugin see
 Test coverage reports
 ---------------------
 
-Astropy can use `coverage.py <https://coverage.readthedocs.io/en/latest/>`_ to
-generate test coverage reports.  To generate a test coverage report, use::
+Coverage reports can be generated using the `pytest-cov
+<https://pypi.org/project/pytest-cov/>`_ plugin by using e.g.::
```

Comment:
Is this a dependency of `pytest-astropy` as well? If so, do mention.

Reply:
Leftover from previous review... Though from `setup.cfg` it is clear it is not a dependency of `pytest-astropy` - find to leave that as is for now.

Reply:
pytest-astropy 0.8 now includes pytest-cov, so I've updated the wording here to mention that this is the case

Reply:
Super!